services/user_api/queries.js

```diff
@@ -0,0 +1,82 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'admin',
+  host: 'localhost',
+  database: 'CS3219',
+  password: 'password',
+  port: 5432,
+})
+
+
+const getUsers = (request, response) => {
+    pool.query('SELECT * FROM users ORDER BY userId ASC', (error, results) => {
```

Comment:
```suggestion
    pool.query('SELECT username, email, createdDateTime FROM users ORDER BY userId ASC', (error, results) => {
```
I think maybe shouldn't return password to frontend

Reply:
Agreed.